addons/project/static/tests/tours/project_task_history.js

```diff
@@ -112,6 +112,41 @@ registry.category("web_tour.tours").add("project_task_history_tour", {
                 throw new Error(`Expect description to be ${expected}, got ${p}`);
             }
         }
+    }, {
+        content: "Open tab Tasks.",
+        trigger: 'button[data-menu-xmlid="project.menu_project_management"]',
+        async run() {
+            $('button[data-menu-xmlid="project.menu_project_management"]').click();
+            await new Promise((r) => setTimeout(r, 1000));
+        },
+    }, {
+        content: "Open tab All Tasks.",
+        trigger: 'a[data-menu-xmlid="project.menu_project_management_all_tasks"]',
+        async run() {
+            $('a[data-menu-xmlid="project.menu_project_management_all_tasks"]').click();
+            await new Promise((r) => setTimeout(r, 1000));
+        },
+    }, {
+        content: "Create a new task.",
+        trigger: 'button.o_list_button_add',
+        async run() {
+            $('button.o_list_button_add').click();
```

Comment:
Don t use jquery, it's depreciated

Reply:
Ok, i search for other test tour and see they 're still using it

Reply:
It's being removed

Reply:
Yes i removed